llmfoundry/models/utils/param_init_fns.py

```diff
@@ -88,8 +86,7 @@ def generic_param_init_fn_(
         )
 
     if init_div_is_residual is not False:
-        if verbose > 1:
-            warnings.warn(
+        log.info(
                 f'Initializing _is_residual layers then dividing them by {div_is_residual:.3f}. ' +\
                 f'Set `init_div_is_residual: false` in init config to disable this.'
             )
```

Comment:
```suggestion
        log.info(
            f'Initializing _is_residual layers then dividing them by {div_is_residual:.3f}. ' +\
            f'Set `init_div_is_residual: false` in init config to disable this.'
        )
```

there are a few instances where you leave the extra indent